docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Report
+    >>> report = Report(api, None, {"description": "Unsigned processes impersonating browsers",
+    ...                             "iocs_v2": [{
+    ...                                 "id": "unsigned-chrome",
+    ...                                 "match_type": "query",
+    ...                                 "values": [("process_name:chrome.exe NOT process_publisher_state:FILE_SIGNATURE_STATE_SIGNED")]
+    ...                                 }],
+    ...                             "link": None,
+    ...                             "severity": 5,
+    ...                             "tags": ["compliance", "unsigned browsers"],
+    ...                             "timestamp": 1601326338,
+    ...                             "title": "Unsigned Browsers"})
+    >>> report.save_watchlist()
+
+Reports should always be given a ``title`` that's sufficiently unique within your organization, so as to minimize
+the chances of confusing two or more Reports with each other.  Carbon Black Cloud will generate unique ``id`` values
+for each report, but does not enforce any uniqueness constraint on the ``title`` of reports.
+
+Alternatively, we can update an existing report, adding more IOCs and/or replacing existing ones.  Note that we have
+to find a report associated with a watchlist by looking in its ``reports`` collection.
+
+Adding the Report to a Watchlist
+++++++++++++++++++++++++++++++++
+Now, add the new Report to a new Watchlist:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Watchlist
+    >>> watchlist = Watchlist(api, None, {"alerts_enabled": False,
+    ...                                   "classifier": None,
+    ...                                   "description": "We are on the lookout for any signs of suspicious applications running on our endpoints",
+    ...                                   "name": "Suspicious Applications",
+    ...                                   "report_ids": [report.id],
+    ...                                   "tags_enabled": True})
+    >>> watchlist.save()
+
+**Note:** ``classifier`` should always be ``None``, unless the new watchlist is being subscribed to a feed.
+
+When either ``alerts_enabled`` or ``tags_enabled`` are ``True``, that Watchlist will create data you can act on -
+either alerts or hits, respectively; if both are ``False``, the Watchlist is effectively disabled.
+
+If you already have an existing Watchlist you wish to enhance, you can add Reports to the existing Watchlist.
+
+Enabling Alerting on a Watchlist
+++++++++++++++++++++++++++++++++
+Once you have the Watchlist configured with the IOCs that are generating the kinds of hits (results) you are after,
+you can enable Alerting for the Watchlist.
+
+::
+
+    >>> watchlist.enable_alerts()
+
+A Closer Look at IOCs
+---------------------
+In this document, we will only discuss the "v2" IOCs; other types of IOCs are only provided for backwards compatibility
```

Comment:
Be clear that `other types of IOCs` is referencing the v1 IOC and those are deprecated and will be converted to v2 IOCs on the API

Reply:
Rephrasing this.